.github/workflows/manual-deploy-l1network.yml

```diff
@@ -58,15 +56,16 @@ jobs:
         uses: 'azure/aci-deploy@v1'
         with:
           resource-group: ${{ secrets.RESOURCE_GROUP }}
-          dns-name-label: testnet-obscuro${{ github.run_number }}
+          dns-name-label: testnet-gethnetwork-${{ github.run_number }}
           image: ${{ secrets.REGISTRY_LOGIN_SERVER }}/obscuronet/obscuro_testnet_l1network:latest
           registry-login-server: ${{ secrets.REGISTRY_LOGIN_SERVER }}
           registry-username: ${{ secrets.REGISTRY_USERNAME }}
           registry-password: ${{ secrets.REGISTRY_PASSWORD }}
-          name: testnet-app
+          name: testnet-gethnetwork-app
           location: 'uksouth'
```

Comment:
@otherview We can't automate this to "closest" or something?

Reply:
Instead of forcing `uksouth`, I mean.

Reply:
I don't think that works that way.. I'll DM you there.